.github/workflows/release-embedding-sdk.yml

```diff
@@ -214,12 +200,24 @@ jobs:
           app-id: ${{ secrets.METABASE_BOT_APP_ID }}
           private-key: ${{ secrets.METABASE_BOT_APP_PRIVATE_KEY }}
 
-      - name: Edit PR adding useful information
+      - name: Edit PR adding useful information (using Metabase bot app)
         run: |
           gh pr edit --add-reviewer @metabase/embedding --add-label no-backport
         env:
           GH_TOKEN: ${{ steps.app-token.outputs.token }}
 
+      - name: Auto approve PR (using GitHub Actions account)
```

Comment:
I tested these 2 step separately, so we don't accidentally merge the PR to master.

Here's the [test for this step.](https://github.com/metabase/metabase/pull/43814#pullrequestreview-2104227524)

You can see that the auto approval works.